src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewEvidenceNotesViewModelMapTests.cs

```diff
@@ -134,9 +134,11 @@ public void Map_GivenValidEvidenceNoteData_ShouldContainHintItems()
             result.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note");
             result.HintItems.ElementAt(0).Value.Should().BeNull();
             result.HintItems.ElementAt(1).Key.Should().Be("Reject evidence note");
-            result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
+            //result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
```

Comment:
@GuidoWorkDev  Remove all of the commented out code

Reply:
whoopsey